docs/source/sensors.rst

```diff
@@ -125,7 +125,7 @@ Timer sensor
 ^^^^^^^^^^^^
 
 Look at the timer sensor implementation
-`here <https://github.com/StackStorm/st2/blob/master/st2reactor/st2reactor/contrib/sensors/st2_timer_sensor.py>`__.
+`here <https://github.com/StackStorm/st2/blob/master/st2reactor/st2reactor/timer/base.py>`__.
```

Comment:
Heh, it was lakshmi. Anyway, it doesn't matter, I just like to give Lakshmi a hard time sometimes for not writing docs :P

#961